nss-day-cohort-48/rare-api-orchestra-outlaws
# consolidated migrations to one file, added new fixtures to rebuild-da…

Consolidated migrations to one file `0001_initial.py`
Added new fixtures comments.json and tags.json to rebuild-database script

STEPS TO TEST
1. fetch this branch and serve locally
2. in vscode delete db.sqlite3
3. run ./rebuild-database
4. all models should be successfully migrated to the new db.sqlite3 file
5. all fixtures should successfully be installed to the new db.sqlite3 file
